serpy/serializer.py

```diff
@@ -83,16 +123,19 @@ class FooSerializer(Serializer):
     """
     #: The default getter used if :meth:`Field.as_getter` returns None.
     default_getter = operator.attrgetter
+    default_setter = attrsetter
 
-    def __init__(self, obj=None, many=False, **kwargs):
+    def __init__(self, obj=None, data=None, many=False, **kwargs):
```

Comment:
I don't particularly like overriding parameters that way: `Serializer(instance_or_data=foo)`. But - I agree that by making a serializer do only one of serialization or deserialization would make it faster (eg. don't need to compile "write fields")